packages/office-ui-fabric-react/src/components/ComponentExamples.test.tsx

```diff
@@ -79,7 +79,9 @@ const excludedExampleFiles: string[] = [
   'Picker.CustomResult.Example.tsx',
   'ScrollablePane.Default.Example.tsx',
   'ScrollablePane.DetailsList.Example.tsx',
-  'SelectedPeopleList.Basic.Example.tsx'
+  'SelectedPeopleList.Basic.Example.tsx',
+  // Snapshots of these examples are worthless since the component isn't open by default
+  'Panel.'
```

Comment:
Confused by this line - why isn't it ".Examples.tsx" like the rest?

Reply:
It's so this will match all panel examples, not just one specific file